RadarSDK/RadarLocationManager.m

```diff
@@ -585,10 +587,11 @@ - (void)replaceSyncedGeofences:(NSArray<RadarGeofence *> *)geofences {
             }
         }
     }
-
-    [RadarNotificationHelper removePendingNotificationsWithCompletionHandler: ^{
-        [RadarNotificationHelper addOnPremiseNotificationRequests:requests];
-    }];
+    if (NSClassFromString(@"XCTestCase") == nil) {
```

Comment:
this method is now ran in our test suite, so we need to suppress this chunk in tests as test env does not have a notification manager.